src/main/java/frc/robot/commands/LimelightUpDown.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.RobotContainer;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import frc.robot.subsystems.LimelightSubsystem;
+
+public class LimelightUpDown extends CommandBase {
+  /** Creates a new LimelightFindTarget. */
+  public double beginningYPosition;
+  public final DrivetrainSubsystem m_DrivetrainSubsystem;
+
+  public LimelightUpDown(DrivetrainSubsystem drivetrainSubsystem) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    m_DrivetrainSubsystem = drivetrainSubsystem;
+    addRequirements(m_DrivetrainSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_DrivetrainSubsystem.setRaw(0.0, 0.0);
+    beginningYPosition = RobotContainer.m_LimelightSubsystem.getTy();
```

Comment:
You should NOT be trying to control a subsystem by calling back thru RobotContainer!

The subsystems you need should be passed into the Command as inputs, saved in private members and then referenced via private members of the Command.

This also applies to all methods in this file.